### PR TITLE
add sum and minimum in mathematical functions to numpy frontend tasks of #1525

### PR DESCRIPTION
**Pull request by AmeenUrRehman**

### CODE REVIEW

**Review by AmeenUrRehman — COMMENTED** [review]
@juliagsy Hey could we ready to merge this

File: ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py [hunk 1]
```diff
@@ -26,3 +26,24 @@ def prod(
 
 
 prod.unsupported_dtypes = {"torch": ("float16",)}
+
+
+# sum
+def sum(
+        x,
```
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py:33]: please follow the signature from numpy docs

from numpy:
`numpy.sum(a, axis=None, dtype=None, out=None, keepdims=<no value>, initial=<no value>, where=<no value>)`

replace initial <no value> with None, and where <no value> with the default True

File: ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py [hunk 2]
```diff
@@ -26,3 +26,24 @@ def prod(
...
+        where=None
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    if ivy.is_array(where):
+        x = ivy.where(where, x, ivy.default(out, ivy.zeros_like(x)), out=out)
+    ret = ivy.sum(x, axis=axis, keepdims=keepdims, out=out)
+    return ret
+
+sum.unsupported_dtypes = {"torch": ("float16",)}
```
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py:49]: this is supported, please remove this line

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py [hunk 3]
```diff
@@ -61,3 +61,56 @@ def test_numpy_prod(
...
+    np_frontend_helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="sum",
+        x=np.asarray(x, dtype=input_dtype[0]),
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py:108]: the arguments might be required to be updated accordingly

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py [hunk 4]
```diff
@@ -61,3 +61,56 @@ def test_numpy_prod(
...
+        axis=axis,
+        dtype=dtype,
+        out=None,
+        keepdims=keepdims,
+        initial=initial,
+        where=where,
+        test_values=True,
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py:115]: please use `test_values=False` as values are tested differently (handled internally) with the presence of `where` argument

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there, great work so far! a few changes are still required and they're stated in comments below! Please request my review again when you're done, thank you!

**Comment by AmeenUrRehman** [comment]
@juliagsy Hey thanku for your review I try to resolve issues could plz review again

**Comment by AmeenUrRehman** [comment]
@juliagsy Check this and thank you

**Comment by AmeenUrRehman** [comment]
@juliagsy plz check the request

**Comment by juliagsy** [comment]
It looks good to go! Did you make issues for your chosen functions? can you mention the issue numbers so that I can link them and merge the pr? thanks!

**Comment by AmeenUrRehman** [comment]
@juliagsy Hey thanks actually the issues no. are #3690 and #3691

**Comment by AmeenUrRehman** [comment]
Hey @juliagsy Thank you for your guidance actually  Ireally forgot to make issue as it is taken by someone else I've to close this pr and will work on another again thanku for your guidance.

**Pull request closed without merging**